src/Akeneo/Pim/Enrichment/Component/Product/Grid/ReadModel/Rows.php

```diff
@@ -16,14 +16,18 @@ final class Rows
     /** @var int */
     private $totalCount;
 
-    /**
-     * @param Row[] $rows
-     * @param int   $totalCount
-     */
-    public function __construct(array $rows, int $totalCount)
+    /** @var int|null */
+    private $productCount;
```

Comment:
```suggestion
    private $totalProductCount;
```

Otherwise, it could be the number of products for this page. Or the other solution is to add phpdoc to explain that (and missing it for `totalCount` also).